scripts/kubernetes/certificate-management.sh

```diff
@@ -0,0 +1,50 @@
+# Collection of functions to add components to manage certificates in a Kubernetes cluster
+
+#######################################
+# Install cert-manager and wait for the the application to be running
+# Arguments:
+#   cert-manager version
+# Examples:
+#   k8s_install_certmanager "v1.11.0"
+#######################################
+k8s_install_certmanager() {
+  local version=$1
+
+  echo "Installing cert-manager..."
+  helm repo add jetstack https://charts.jetstack.io
+  helm repo update
+  kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${version}/cert-manager.crds.yaml
+  helm upgrade --install cert-manager jetstack/cert-manager \
+    --namespace cert-manager --create-namespace \
+    --version ${version} \
+    2>/dev/null
+  kubectl wait pods -n cert-manager -l app.kubernetes.io/instance=cert-manager --for condition=Ready 2>/dev/null
+}
+
+#######################################
+# Create certificate cluster issuers using Let's Encrypt
+# Arguments:
+#   Ingress class name (traefik, nginx, etc.)
+#   administrator email address (to receive notifications for Let's Encrypt)
+# Examples:
+#   k8s_create_letsencryptclusterissuer traefik john.wick@google.com
+#######################################
+k8s_create_letsencryptclusterissuer() {
+  local ingressClassname=$1
+  local emailAddress=$2
+
+  echo "Creating certificate issuers using Let's Encrypt..."
+  helm repo add devpro https://devpro.github.io/helm-charts
```

Comment:
This one we might want to move at some point in time

Reply:
Yes! Added a TODO, really need to do this one (as for the container images)